airbyte-integrations/connectors-performance/destination-harness/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM airbyte/integration-base-java:dev AS build
+WORKDIR /airbyte
+
+ENV APPLICATION destination-harness
+
+COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
+
+RUN tar xf ${APPLICATION}.tar --strip-components=1 && rm -rf ${APPLICATION}.tar
+
+FROM airbyte/integration-base-java:dev
+ARG TARGETARCH
+WORKDIR /airbyte
+
+ENV APPLICATION destination-harness
+# Kubectl and socat are needed in order to create a pod similar to cloud orchestrator
+# It brings up in cluster the destination pod to measure, and communicates over socat
+RUN curl -O https://s3.us-west-2.amazonaws.com/amazon-eks/1.25.6/2023-01-30/bin/linux/${TARGETARCH}/kubectl
+RUN chmod +x ./kubectl
+RUN mv ./kubectl /bin/
+RUN yum install -y socat && yum clean all
+
+COPY --from=build /airbyte /airbyte
+COPY base.sh .
```

Comment:
Rodi, remind me again, why do we need this?

Reply:
This is on the wrong line, so I'm resolving.

Reply:
the base.sh is entry point of this container. Inherited from [/airbyte-integrations/integration-base](https://github.com/airbytehq/airbyte/blob/2d7180606ecb809d8613e231627b48b9e001292f/airbyte-integrations/bases/base/Dockerfile#L8):
```
ENTRYPOINT ["/airbyte/base.sh"]
```
We implement our own base.sh since the harness needs different argument than connector (no --read or --write or --state for example).
compare to the [original base.sh](https://github.com/airbytehq/airbyte/blob/2d7180606ecb809d8613e231627b48b9e001292f/airbyte-integrations/bases/base/base.sh)